waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -96,3 +98,18 @@ generatePrismaClient genProjectRootDirAbs = do
       writeDbSchemaChecksumToFile genProjectRootDirAbs (SP.castFile dbSchemaChecksumOnLastGenerateFileProjectRootDir)
       return $ Right ()
     ExitFailure code -> return $ Left $ "Prisma client generation failed with exit code: " ++ show code
+
+-- | Checks `prisma migrate diff` exit code to determine if schema.prisma is
+-- different than the DB. Returns Nothing on error as we do not know the current state.
+doesSchemaMatchDb :: Path' Abs (Dir ProjectRootDir) -> IO (Maybe Bool)
```

Comment:
Does this one care about migrations? I remember we cared about schema <> migrations <> db relationship but I can't remember exactly how this PR now fits into it, could you super quickly remind me?

Reply:
ok, so this one doesn't care about migrations! Well makes sense, its name says it clearly. Maybe we can mention it additionally in comments just to be super clear, but if we dont its ok.

Reply:
Yeah I can add. More context always helps! 👍🏻